api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -547,9 +547,9 @@ message HttpConnectionManager {
   // race with the final GOAWAY frame. During this grace period, Envoy will
   // continue to accept new streams. After the grace period, a final GOAWAY
   // frame is sent and Envoy will start refusing new streams. Draining occurs
-  // both when a connection hits the idle timeout or during general server
-  // draining. The default grace period is 5000 milliseconds (5 seconds) if this
-  // option is not specified.
+  // for a connection when there are no active streams and the idle timeout
```

Comment:
Are you just adding this for more clarity because if connection hits idle timeout that means there are no active streams?

Reply:
Yes, I initially wanted to clarify that connections with active streams won't become idle, but I realized that the idle_timeout documentation already explains this clearly.